File: Forge/src/main/java/siongsng/rpmtwupdatemod/mixin/ChatScreenAccessor.java
package siongsng.rpmtwupdatemod.mixin;

import net.minecraft.client.gui.components.EditBox;
import net.minecraft.client.gui.screens.ChatScreen;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

public class ChatScreenAccessor {
    @Mixin(ChatScreen.class)
    public interface chatFieldAccessor {
        @Accessor("input")
        EditBox getChatField();
    }

}

File: common/src/main/kotlin/com/rpmtw/rpmtw_platform_mod/mixins/ChatScreenAccessor.kt
@file:JvmName("ChatScreenAccessor")

package com.rpmtw.rpmtw_platform_mod.mixins

import net.minecraft.client.gui.components.EditBox
import net.minecraft.client.gui.screens.ChatScreen
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

class ChatScreenAccessor {
    @Mixin(ChatScreen::class)
    interface ChatFieldAccessor {
        @get:Accessor("input")
        val chatField: EditBox?
    }
}